sycl/doc/extensions/supported/sycl_ext_intel_fpga_device_selector.asciidoc

```diff
@@ -92,16 +92,24 @@ supports.
 |fpga_simulator_selector added.
 |===
 
+|2
+|SYCL 2020 selector variants `fpga_selector_v`, `fpga_simulator_selector_v`, and
+`fpga_emulator_selector_v` added.  Old selectors `fpga_selector`,
+`fpga_simulator_selector`, and `fpga_emulator_selector` deprecated.
```

Comment:
The HTML render is broken here.  Click "view file" in this PR to see what the render looks like.
I'm not sure why, but maybe introducing a leading space like this will help?

```suggestion
 `fpga_emulator_selector_v` added.  Old selectors `fpga_selector`,
 `fpga_simulator_selector`, and `fpga_emulator_selector` deprecated.
```

Reply:
Oh, I see the problem now.  You have an extra `|===`.